tests/rkt_api_service_test.go

```diff
@@ -326,8 +336,12 @@ func TestAPIServiceListInspectPods(t *testing.T) {
 	defer os.Remove(manifestFile)
 
 	runCmd := fmt.Sprintf("%s run --pod-manifest=%s", ctx.Cmd(), manifestFile)
-	esp := spawnOrFail(t, runCmd)
-	waitOrFail(t, esp, 0)
+	waitOrFail(t, spawnOrFail(t, runCmd), 0)
+
+	gcCmd := fmt.Sprintf("%s gc --mark-only=true", ctx.Cmd())
+	waitOrFail(t, spawnOrFail(t, gcCmd), 0)
+
+	gcTime := time.Now()
```

Comment:
Since compareTime checks if the times are equal within a 2 second precision, I think the test would be more meaningful if we wait some time before marking the pod. In the rkt list test [we wait 4 seconds](https://github.com/coreos/rkt/blob/84e07dded413237072a8477b67237f8d95ad715a/tests/rkt_list_test.go#L27).